portfolio/src/main/java/com/google/sps/servlets/ListCommentServlet.java

```diff
@@ -0,0 +1,96 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.*;
+import com.google.gson.Gson;
+import com.google.sps.data.Comment;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.google.sps.utils.Constants.COMMENT_KEY;
+import static com.google.sps.utils.Constants.COMMENT_MAXNUMBER;
+import static com.google.sps.utils.Constants.COMMENT_TIMESTAMP;
+
+
+/** Servlet that handles posting comment content. */
+@WebServlet("/list-comment")
+public class ListCommentServlet extends CommentServlet {
```

Comment:
What's the different between list comment servlet and comment servlet? Is it worth splitting to two level of inheritance for just 1 static method?

Reply:
Fixed